variables.tf

```diff
@@ -222,6 +222,7 @@ variable "kms_key_deletion_window" {
 
 # Secrets Manager
 # ---------------
+
 variable "tfe_license_secret" {
```

Comment:
Each of the secret variables in the root module and the submodules need to be updated to `type = string` in order to align with the outputs of the secrets fixture.